tests/devapps/WAM/NetFrameworkWam/Form1.cs

```diff
@@ -75,7 +76,12 @@ private IPublicClientApplication CreatePca()
                 .Create(clientId)
                 .WithAuthority(this.authorityCbx.Text)
                 .WithExperimentalFeatures(true)
-                .WithBroker(this.useBrokerChk.Checked)
+
+                //.WithWindowsBroker(true)
+                .WithWindowsBroker(true)
+
+                //.WithBroker(true)
```

Comment:
>//.WithBroker(true) [](start = 16, length = 19)

naming: would have been better if we could stick to this terminology, what will we do on Linux/Mac when we have a broker there?
Seems unfortunate that we can't have both names the same?

Reply:
We should discuss about this in more detail, I only discussed it with @jmprieur . There are 2 aspects: 

1. discoverability of API - if we hide `WithBroker` in MSAL and only make it available on the extension, it is harder to discover. It would also be a (small) breaking change, as we have WithBroker API today, even if it simply throws.
2. netstandard support - from a netstandard perspective, we already have WithBroker (to support Xamarin). 

It's an interesting question this - how will we support Mac and Linux brokers? I imagine we'll only be able to PInvoke into such brokers, so they will live in MSAL package. 



```csharp
// net5 and UWP
.WithBroker(true);

// net core 3.x and net classic
.WithWindowsBroker(true)
.WithBroker(true)
```